gtsfm/utils/tracks.py

```diff
@@ -30,7 +29,7 @@ def classify_tracks2d_with_gt_cameras(
         The triangulation exit code for each input track, as list of the same length as of tracks.
     """
     # do a simple triangulation with the GT cameras
-    cameras_dict: Dict[int, Cal3Bundler] = {i: cam for i, cam in enumerate(cameras_gt)}
+    cameras_dict: Dict[int, PinholeCameraCal3Bundler] = {i: cam for i, cam in enumerate(cameras_gt)}
```

Comment:
`Point3dInitializer` requires the updated input type: https://github.com/borglab/gtsfm/blob/master/gtsfm/data_association/point3d_initializer.py#L130